### PR TITLE
fix(saas): apply new operate config

### PR DESCRIPTION
**Pull request by chillleader**
## Description

Applies the necessary changes to run Connectors in SaaS with the latest Spring Zeebe version.

## Related issues

closes https://github.com/camunda/team-connectors/issues/751

### CODE REVIEW

**Review by ev-codes — APPROVED** [review]
Thank you!

File: bundle/camunda-saas-bundle/src/test/java/io/camunda/connector/runtime/saas/TestSpringContextStartup.java [hunk 1]
```diff
@@ -48,17 +46,4 @@ public void contextLoaded() {
     // when the Spring context is initialized (e.g.
     // https://github.com/camunda/team-connectors/issues/251)
   }
-
-  @Test
-  public void operatePropertiesAreSet() {
```
Comment (ev-codes) [inline bundle/camunda-saas-bundle/src/test/java/io/camunda/connector/runtime/saas/TestSpringContextStartup.java:53]: Should we replace these with a new test?

Perhaps one that checks the JWT credential is configured correctly?

Comment (chillleader) [reply bundle/camunda-saas-bundle/src/test/java/io/camunda/connector/runtime/saas/TestSpringContextStartup.java:53]: Done!

**Review by ev-codes — APPROVED** [review]

**Pull request merged**